### PR TITLE
moneymanagerex: Add version 1.5.14

### PR DESCRIPTION
**Pull request by ClassicDarkChocolate**
[MoneyManager Ex](https://moneymanagerex.org/) is a open-source, cross-platform, software that helps you organize your finances and keep track of where, when and to who the money goes.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/moneymanagerex.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "version": "1.5.14",
+    "description": "A free, cross-platform personal finance software",
+    "homepage": "https://moneymanagerex.org/",
+    "license": "GPL-2.0",
```
Comment (rashil2000) [inline bucket/moneymanagerex.json:5]:
```suggestion
    "license": "GPL-2.0-only",
```

**Comment by rashil2000** [comment]
Where are the configuration files stored?

**Comment by ClassicDarkChocolate** [comment]
> Where are the configuration files stored?

@rashil2000 It seems that it store its configuration file in [%APPDATA%\MoneyManagerEx\mmexini.db3](https://moneymanagerex.org/moneymanagerex/docs/en/index.html#section19.5)

**Comment by rashil2000** [comment]
> > Where are the configuration files stored?
> 
> @rashil2000 It seems that it store its configuration file in [%APPDATA%\MoneyManagerEx\mmexini.db3](https://moneymanagerex.org/moneymanagerex/docs/en/index.html#section19.5)

For both .exe and .zip? That seems rather odd, portable apps usually store them in the installation folder itself.

**Comment by ClassicDarkChocolate** [comment]
> > > Where are the configuration files stored?
> > 
> > 
> > @rashil2000 It seems that it store its configuration file in [%APPDATA%\MoneyManagerEx\mmexini.db3](https://moneymanagerex.org/moneymanagerex/docs/en/index.html#section19.5)
> 
> For both .exe and .zip? That seems rather odd, portable apps usually store them in the installation folder itself.

It will create mmexini.db3 in %APPDATA% after user launch the program, but I don't think that configuration file is important though. They let user specify where to store accounting data as *.mmb (like *.gnucash or *.xlsx).

**Pull request merged**